interface/new/new_comprehensive.php

```diff
@@ -382,27 +382,34 @@ function validate(f) {
         <?php } ?>
         var msg = "";
         var ins_error = '0';
-        msg += "<?php echo htmlspecialchars(xl('The following fields are required'),ENT_QUOTES); ?>:\n\n";
+        msg += "<?php echo htmlspecialchars(xl('The following fields are required'),ENT_QUOTES); ?>:</br>";
         for ( var i = 0; i < errMsgs.length; i++ ) {
-               msg += errMsgs[i] + "\n";
+               msg += errMsgs[i] + "<br>";
         }
         <?php if($GLOBALS['primary_insurance_required']){ ?>
           if($('#i1provider option:selected').val() == '') {
              var ins_error = '1';
-             msg += "<?php echo htmlspecialchars(xl('Primary Insurance Provider Entry Required.'),ENT_QUOTES); ?> \n";
+             msg += "<?php echo htmlspecialchars(xl('Primary Insurance Provider Entry Required.'),ENT_QUOTES); ?> </br>";
           }
         <?php } ?>
-        msg += "\n<?php echo htmlspecialchars(xl('Please fill them in before continuing.'),ENT_QUOTES); ?>";
+        msg += "</br><?php echo htmlspecialchars(xl('Please fill them in before continuing.'),ENT_QUOTES); ?>";
 
         if ( errMsgs.length > 0 || ins_error == '1' ) {
-               alert(msg);
-               return false;
+          iziToast.error({
+            title: 'Error',
+            message: msg,
+            position: 'topRight'
+          });
```

Comment:
@punwai I will prefer each message on a new line. Not a bacth of them look for a way to separate them